### PR TITLE
Create person.schema.json

### PR DESCRIPTION
**Pull request by skoehnen**
Described in #83

### CODE REVIEW

File: v3.0/products/person.schema.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "$id": "https://openminds.ebrains.eu/core/person.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Researcher that was involved in producing the research products or their versions.",
```
Comment (lzehl) [inline v3.0/products/person.schema.json]: "... the research product or it's versions."

File: v3.0/products/person.schema.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type",
+    "@id",
+    "email"
```
Comment (lzehl) [inline v3.0/products/person.schema.json]: "givenName" should be also required. Sorry if that was not documented

File: v3.0/products/person.schema.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "familyName": {
+      "description": "Family name of this person.",
+      "type": "string"
+    },
+    "givenName": {
+      "description": "Given name of this person.",
+      "type": "string"
+    },
+    "digitalIdentifier": {
+      "description": "Globally unique and persistent digital identifier for the person.",
```
Comment (lzehl) [inline v3.0/products/person.schema.json]: "... for this person."

**Review by lzehl — CHANGES_REQUESTED** [review]
just some small adjustments needed

File: v3.0/products/person.schema.json (hunk 1 above)
Comment (skoehnen) [reply v3.0/products/person.schema.json]: Fixed

File: v3.0/products/person.schema.json (hunk 2 above)
Comment (skoehnen) [reply v3.0/products/person.schema.json]: It was documented :+1: I missed it.

File: v3.0/products/person.schema.json (hunk 3 above)
Comment (skoehnen) [reply v3.0/products/person.schema.json]: I updated

**Review by lzehl — APPROVED** [review]

**Pull request merged**